### DISCUSSION TITLE
UK based interest?

### DISCUSSION THREAD

**Discussion by dmgeurts** [tjko/fanpico#21] [General]
Looking to get a 0804D printed and wondering if there are others interested, maybe we can split the cost. Sadly the EU market is now closed to us (import duties are risky/costly) so we'll need to find a local supplier.

**Comment by dmgeurts** [comment]
And how much would people be willing to pay for a PCB? I could order a batch and list them on eBay or here (if permitted) for a fixed price.

**Reply by tjko** [reply]
That would be fine.  Fanpico is under GPL license, but I guess that probably doesn't translate well to PCB designs.  PCBs have Open Source Hardware logo and intent to be "free".   Maybe need to add some "OSH" license reference regarding that...

**Reply by dmgeurts** [reply]
> That would be fine. Fanpico is under GPL license, but I guess that probably doesn't translate well to PCB designs. PCBs have Open Source Hardware logo and intent to be "free". Maybe need to add some "OSH" license reference regarding that...

Yeah, that's a consideration. I'm not planning to make money off anything, but ordering PCBs isn't free. I still need to check the cost of components if I sourced those separately. But as I don't have an SMD heat gun, I'm tempted by ordering units with all SMD components preassembled.

Ordering larger batches will bring costs down, hence fishing for local interest to see how far I can bring the cost per unit down. I'd rather not get stuck with 8 PCBs that I have no need for.

**Comment by dmgeurts** [comment]
Including assembly, a batch of 10 could be ordered from JLCPCB for just under £18 each. Excluding Pico and connectors.

**Comment by tjko** [comment]
I still should have few of the 0804D v1.0 PCBs that PCBway sponsored to 'giveaway'.  I've sent one PCB to UK in plain envelope and it went through fine, so if anyone in UK wants one I can send one if you cover the stamps (about $3), while I still have PCBs left...

(only major difference between v1.0 and v1.1 is the cutout for Pico W antenna....)

**Reply by dmgeurts** [reply]
@tjko with the Pico W, can a fanpico function as a standalone IoT device? I have some rack fans I build a circuit for controlled by a RPi Zero, but my circuit had some flaws in it and I fried a couple of Zeros... (Oops)

With two temperature sensors and being able to read the tacho of the connected fans, it's perfect for what I need it to do. I'd like to read the values so I can ingest them into InfluxDB and graph the values using Grafana. In both my use cases I'll need to set the fan curves purely based on the highest reading from one or more connected sensors.

**Reply by tjko** [reply]
Current firmware provides basic http server with simple "UI" to view status with a browser. And there is also URLs to get status in JSON and CSV formats for automated monitoring, etc...

At the moment there is no way to configured fanpico over network (USB or serial TTL connection required for that). But maybe could add option to enable TCP port for 'console' access over network....

**Comment by dmgeurts** [comment]
I've just ordered 10 0804D PCBs, 5 with SMD parts assembled and 5 bare. I need two so I'll have 8 PCBs available in the UK soon. Let me know if you'd like one if you're based in the UK.

3x £18 with all SMD parts assembled
5x £5 for the bare PCB
(excl Shipping)

PCBs are black with white silk screen.
